src/components/Loader/Loader.jsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react'
+import * as S from './Style'
+
+const Loader = () => {
+  return(
+    <S.Wrapper><div></div><div></div><div></div><div></div><div></div></S.Wrapper>
```

Comment:
WTF?!

Reply:
Loader tak wyglada

Reply:
zmienmy to pozniej na cos normalnego, prosta animacja w cssach?

Reply:
tak, pozniej pewnie robimy to ikoną czy czymś innym